types/react-native/index.d.ts

```diff
@@ -8651,8 +8662,14 @@ export interface KeyboardStatic extends NativeEventEmitter {
 export const ART: ARTStatic;
 export type ART = ARTStatic;
 
-export const SectionList: SectionListStatic<any>;
-export type SectionList<ItemT> = SectionListStatic<ItemT>;
+export const ImagePickerIOS: ImagePickerIOSStatic;
+export type ImagePickerIOS = ImagePickerIOSStatic;
+
+export const LayoutAnimation: LayoutAnimationStatic;
+export type LayoutAnimation = LayoutAnimationStatic;
+
+export const Systrace: SystraceStatic;
+export type Systrace = SystraceStatic;
```

Comment:
Why's this adding new constants? This doesn't have anything to do with the PR description. This is why the tests are failing because these are already declared and you're declaring them again here.

https://travis-ci.org/DefinitelyTyped/DefinitelyTyped/builds/603212892#L3867-L3878

Reply:
I have no clue where they're coming from - maybe from some rebase I did? Anyway, I removed them.